modules/ROOT/pages/oas3-support.adoc

```diff
@@ -0,0 +1,88 @@
+* Share OAS 3.0 API specifications.
+
+[NOTE]
+====
+If you want to share an OAS 3.0 API specification as OAS 2, download your OAS 3.0 API specification from Anypoint Exchange as-is, and then convert it to OAS 2.0 by using a tool such as APIMatic transformer.
+====
+
+* View documentation that is written into OAS 3.0 API specifications.
+* Publish OAS 3.0 API specifications to API Community Manager.
+* Download OAS 3.0 API specifications as RAML API specifications for scaffolding and implementation with 3rd party IDEs.
++
+To make an OAS 3.0 API specification available for downloading as RAML from Anypoint Exchange for use in Anypoint Studio and API Manager:
++
+. Import the OAS 3.0 API specification into API Designer.
+. Click the three dots next to the filename in the *Files* pane and select *Duplicate*.
+. In the *Duplicate As* dialog, select *RAML*.
+. Set the root file of the project to the RAML version of the API specification.
+. Publish the project to Anypoint Exchange.
+
+
+
+
+== Beta Features in API Community Manager
+
+* Publish OAS 3.0 specifications to your API community, sharing them with partners and customers to help drive their adoption.
+* View documentation that is written into OAS 3.0 specifications.
+
+== Beta Features in API Designer
+
+* Create API specifications in OAS 3.
++
+When creating a new API-specification project, select *OAS 3.0 (Beta)* in the *Language* field.
+* Import OAS 3.0 specifications from Anypoint Exchange or from your file system.
++
+.How to import files into an API-specification project in API Designer
+image::apid-import.png[Options for importing]
++
+. Click the gear icon in the top-right corner of the text editor.
+. Select either of these options:
+** Select *Import from Exchange* to see a list of the API specifications that are available from the business organization that your user ID belongs to in Anypoint Platform and a list of the API specifications that are published by MuleSoft.
+** Select *Import* to import an API specification from your local filesystem.
+* Develop and edit OAS 3.0 specfications.
+* View documentation that is written into OAS 3.0 API specifications.
+* Simulate calls to methods that are in OAS 3.0 API specifications.
+* Publish OAS 3.0 API specifications to Anypoint Exchange.
++
+[NOTE]
+====
+If you want to publish an OAS 3.0 API specification to Anypoint Exchange as a RAML API specification, follow these steps:
++
+. Next to the name of the file, click the three dots.
+. Select *Duplicate As*.
+. In the *Duplicate As* dialog, select *RAMl 1.0*.
+. Set the new duplicate as the root file in the project.
+. Publish the project to Anypoint Exchange.
+====
+* Share OAS 3.0 specifications as RAML specifications with Anypoint Studio and API Manager.
++
+.How to convert OAS 3.0 files to RAML for import into Anypoint Studio or API Manager
+image::apid-duplicate.png[The Download As option]
++
+. Next to the name of the file, click the three dots.
+. Select *Download As*.
```

Comment:
We need to replace "download as" with the "duplicate as" user action to allow users to use from API Manager and Anypoint Studio.